.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  BuildAndDeploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Push backend to Dockerhub
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: dagandersen/backend
+        username: ${{ secrets.DOCKER_DAG_TEST_USER }}
+        password: ${{ secrets.DOCKER_DAG_TEST_PASSWORD }}
+        workdir: backend
+    - uses: actions/checkout@master
+    - name: Push frontend to Dockerhub
+      uses: elgohr/Publish-Docker-Github-Action@master
+      with:
+        name: dagandersen/frontend
+        username: ${{ secrets.DOCKER_DAG_TEST_USER }}
```

Comment:
Shouldn't this be my user account? We can use your account instead if need be 😄

Reply:
Feel free to change it! I couldn't test it since i don't have access to your account.

Reply:
Det kræver du selv opretter en front- og backend container på din dockerhub account... Den kan ikke tilsluttes din account uden de er oprettet.